### PR TITLE
Handle 201 status code from WireMock

### PR DESCRIPTION
**Pull request by bloomonkey**
Accept 201 status when posting new mapping

WireMock standalone (v2.6.0) returns `201 Created` status when
creating new mappings via a POST request.

`RestClient.handle_response()` updated accordingly with supporting
test.

### CODE REVIEW

File: wiremock/tests/base_tests/base_resource_tests.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""TODO: Document base_resource_tests here.
+
+Copyright (C) 2017, Auto Trader UK
```
Comment (platinummonkey) [inline wiremock/tests/base_tests/base_resource_tests.py]: can you remove this copyright and TODO information?

Comment (platinummonkey) [reply wiremock/tests/base_tests/base_resource_tests.py]: there are 2 more as well it looks like

**Comment by platinummonkey** [comment]
thanks for the PR! looking good, just had a couple minor things to take care of before merging and publishing

File: wiremock/tests/base_tests/base_resource_tests.py (hunk 1 above)
Comment (bloomonkey) [reply wiremock/tests/base_tests/base_resource_tests.py]: Yes, sorry. IDE template created these, I'll strip them out

**Comment by platinummonkey** [comment]
Docs and PyPI updated. Thanks!

**Pull request merged**